app/src/main/java/com/nervousfish/nervousfish/modules/pairing/AbstractPairingHandler.java

```diff
@@ -4,12 +4,12 @@
  * Contains common methods shared by all pairing modules.
  */
 @SuppressWarnings("PMD.AbstractClassWithoutAbstractMethod")
-abstract class APairingHandler {
+abstract class AbstractPairingHandler {
```

Comment:
I think it is because you have to type the name of interfaces very often while you generally don't want to type the name of a (abstract) class often